addOns/reports/src/main/zapHomeFiles/reports/traditional-html-plus/report.html

```diff
@@ -518,6 +518,116 @@ <h3 th:text="#{report.alerts.detail}">Alert Detail</h3>
 			<div class="spacer"></div>
 		</th:block>
 	</th:block>
+
+	<th:block th:if="${reportData.isIncludeSection('sequencedetails')}">
+		<th:block
+			th:if="${reportData.reportObjects.get('seqAScanData') != null}">
+			<h3 th:text="#{report.sequences.summary.name}">Summary of
+				Sequences</h3>
+			<sup>For each step: result (Pass/Fail) - risk (of highest
+				alert(s) for the step, if any).</sup>
+
+			<th:block
+				th:each="seq, seqState: ${reportData.reportObjects.get('seqAScanData').seqData}">
+				<table class="alerts">
+					<tr>
```

Comment:
For long sequences we should spread this over multiple lines, eg if a sequence has 60 steps this wouldnt look good?

Reply:
Every 10:
![image](https://github.com/user-attachments/assets/aae3949e-69d2-43d1-9231-55276b410c40)